skylight-hq/phinvads-go
# Load Hot Topics from database

resolves #45 

## What this PR changes:
- adds in xo-generated models from the `hot_topic` table
- adds a Templ template for a single Hot Topic component
- adds an endpoint to fetch all* hot topics and return them to the client
- adds an `hx-trigger` to call above endpoint on home page load 

\* all hot topics, excluding the "System Config" row

## How to test:
- make sure the topics appear on page load
- make sure each topic is able to be clicked to expand/collapse the detail text below it

## Notes:
- many of these topics contain additional links and downloads on the actual PHIN VADS page, but these are not included with the data returned from the database
- actual PHIN VADS page has "Collapse All/Expand All" buttons, but these were not included in the figma mockups
